docs/auth0_users_search.md

```diff
@@ -5,10 +5,21 @@ layout: default
 
 Search for users. To create one, run: `auth0 users create`.
 
+## Usage
 ```
 auth0 users search [flags]
 ```
 
+## Examples
+
+```
+auth0 users search
+  auth0 users search --query id
+  auth0 users search -q name --sort "name:1"
+  auth0 users search -q name -s "name:1"
+  auth0 users search -q name -s "name:1" --json
```

Comment:
Other examples only feature the short version of the flags at the end, in the example that features the `--json` flag.
Maybe we can use `auth0 users search -query name --sort "name:1"` instead of `auth0 users search -q name --sort "name:1"`, and remove `auth0 users search -q name -s "name:1"`?